client/templates/app/nav/nav.html

```diff
@@ -10,11 +10,10 @@
           {{> notificationsButton}}
           {{> userDropdown}}
         {{else}}
-          <li><a href="{{pathFor 'login'}}">Sign in</a></li>
-          <li><a href="{{pathFor 'signup'}}">Register</a></li>
+          <li><a href="/signup">Login / Register</a></li>
```

Comment:
I don't think I like the look of `Login / Register`. Let's do some research and find what existing patterns are out there that we can look at. Twitter has separate `Log in` and `Sign up` buttons. Facebook has somewhat the same idea.

I think we can make this look better. @iamlasse should be able to help with it since he's got a good design eye.